modules/reference/pages/schedule/maintenance-windows.adoc

```diff
@@ -0,0 +1,14 @@
+[[schedule-maint-windows]]
+= Maintenance Windows
+
+The menu:Schedule[Maintenance Windows] section allows you to manage your maintenance window schedules.
+To create a maintenance window, you need to have an existing [path]``.ical`` file that contains the calendar.
+Create a new schedule, and provide a link to the calendar file.
```

Comment:
```suggestion
Create a new schedule, and select the calendar.
```
When creating a *Schedule*, user just selects the *Calendar* that they created above, they don't provide links anymore.